var/spack/repos/builtin/packages/py-traceback2/package.py

```diff
@@ -15,4 +15,5 @@ class PyTraceback2(PythonPackage):
     version('1.4.0', sha256='05acc67a09980c2ecfedd3423f7ae0104839eccb55fc645773e1caa0951c3030')
 
     depends_on('py-setuptools', type='build')
-    depends_on('py-pbr', type='build')
+    depends_on('py-six',        type='build')
```

Comment:
It looks like this is needed at run-time too:
```suggestion
    depends_on('py-six',        type=('build', 'run'))
```